WPThemeReview/Sniffs/Privacy/NoUrlShortenersSniff.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * WPThemeReview Coding Standard.
+ *
+ * @package WPTRT\WPThemeReview
+ * @link    https://github.com/WPTRT/WPThemeReview
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WPThemeReview\Sniffs\Privacy;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Check if the file contains a URL shortener.
+ *
+ * @since 0.2.0
+ */
+class NoUrlShortenersSniff implements Sniff {
+
+	/**
+	 * Error message template.
+	 *
+	 * @var string
+	 */
+	const ERROR_MSG = 'No URL shorteners should used in the theme. Found: "%s".';
+
+	/**
+	 * Found used shortener in a file
+	 *
+	 * @var string
+	 */
+	protected $shortener;
+
+	/**
+	 * List of url shorteners.
+	 *
+	 * @since 0.2.0
+	 *
+	 * @var array
+	 */
+	protected $url_shorteners = [
+		'goo.gl',
```

Comment:
Is there any logic to the order used in this list ? If so, please add a comment to the docblock explaining the order.
If not, please order alphabetically.

Reply:
No order logic, I'll reorder it. :+1: